redbot/cogs/audio/audio.py

```diff
@@ -130,9 +129,12 @@ def __init__(self, bot):
             if notify_channel:
                 notify_channel = self.bot.get_channel(notify_channel)
                 embed = discord.Embed(
-                    colour=notify_channel.guild.me.top_role.colour, title="Queue ended."
+                    title="Queue ended."
                 )
-                await notify_channel.send(embed=embed)
+                if await ctx.embed_requested():
+                    await notify_channel.send(embed=embed)
+                else:
+                    await notify_channel.send(embed=embed)
```

Comment:
This isn't doing anything. The condition statement is meaningless - the bot will send the exact same thing no matter the condition.